lib/Site.js

```diff
@@ -26,25 +26,30 @@ const TEMPLATE_ROOT_FOLDER_NAME = 'template';
 const TEMPLATE_SITE_ASSET_FOLDER_NAME = 'markbind';
 const USER_VARIABLES_PATH = '_markbind/variables.md';
 
+const TITLE_PREFIX_SEPARATOR = ' - ';
+
 const SITE_CONFIG_DEFAULT = {
   baseUrl: '',
-  pages: [
-    {
-      src: 'index.md',
-      title: 'Hello World',
-    },
-  ],
   ignore: [
     '_site/*',
-    '*.json',
+    'site.json',
     '*.md',
   ],
+  pages: [
+    {
+      glob: '**/index.md',
+    },
```

Comment:
Can you add this above (rather than replace here) for a clean diff and to show that both work together?

```diff
  pages: [
    {
      src: 'index.md',
      title: 'Hello World',
    },
+   {
+     glob: '**/index.md',
+   },
  ],
```